### PR TITLE
Initial capification

### LINKED ISSUE #4: Add deployment scripting to repo
**ACCEPTANCE**
- [ ] Team members can deploy to the staging environment on AWS

### PR DESCRIPTION
**Pull request by acozine**
Closes #4.

@bess @mark-dce please confirm that if you check out this branch and do `BRANCH_NAME=capify bundle exec cap aws deploy` the code deploys for you.

### CODE REVIEW

**Comment by acozine** [comment]
@bess: I put `role_map.yml` back into the shared files list - otherwise we'd need to check production auth information into the repo. I added the shared file on the sandbox server, so we're all set.

File: Gemfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ gem 'hyrax', '1.0.0.rc1'
 gem 'jbuilder', '~> 2.5'
 # Use jquery as the JavaScript library
 gem 'jquery-rails'
+gem 'pg'
```
Comment (mark-dce) [inline Gemfile]: I'm slightly curious why pg was added here, I don't see any other references to postgres in config or sample files.

File: Gemfile [hunk 2]
```diff
@@ -23,13 +24,14 @@ gem 'rsolr', '~> 1.0'
...
 gem 'sqlite3'
+gem 'therubyracer'
 # Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
 gem 'turbolinks', '~> 5'
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
-
+gem 'yard'
```
Comment (mark-dce) [inline Gemfile]: Do we need yard in production?  Otherwise, should we put it in a development and test group?

**Comment by mark-dce** [comment]
I was able to run `BRANCH_NAME=capify bundle exec cap was deploy` and everything completed successfully.

**Comment by bess** [comment]
I will try this deploy, and I will try re-working the Gemfile to address @mark-dce's comments

**Comment by mark-dce** [comment]
I went ahead and merged this so it's not sitting.  I opened #85 to add the remainder of the work to put sample Postgres config in the repo.

**Pull request merged**